datamodels/2.x/itop-core-update/dictionaries/en.dict.itop-core-update.php

```diff
@@ -77,6 +78,9 @@
 	'iTopUpdate:UI:CanCoreUpdate:Warning' => 'Warning: application update can fail: %1$s',
 	'iTopUpdate:UI:CannotUpdateUseSetup' => 'You must use the <a href="%1$s">setup</a> to update the application.<br />Some modified files were detected, a partial update cannot be executed.',
 
+	'iTopUpdate:UI:SetupLaunch' => 'Launch iTop Setup',
```

Comment:
```suggestion
	'iTopUpdate:UI:SetupLaunch' => 'Launch '.ITOP_APPLICATION_SHORT.' Setup',
```

Same comment about the application name constant

Reply:
Fixed in 361ffe90